WalletWasabi/WabiSabi/Models/ReissueCredentialRequest.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using WalletWasabi.WabiSabi.Crypto.CredentialRequesting;
+
+namespace WalletWasabi.WabiSabi.Models
+{
+	public record ReissueCredentialRequest(
+		Guid RoundId,
+		RealCredentialsRequest RealAmountCredentialRequests,
+		RealCredentialsRequest RealVsizeCredentialRequests,
+		ZeroCredentialsRequest ZeroAmountCredentialRequests1,
+		ZeroCredentialsRequest ZeroAmountCredentialRequests2
```

Comment:
This should be `ZeroAmountCredentialsRequest` and `ZeroVsizeCredentialRequest`.

Also note that it's called `CredentialsRequest` not `CredentialRequests` apart from in the test factory, we should probably be consistent because otherwise doing searches through the code base is confusing for such a small difference and it's easy to miss things.

FWIW this *only* ensures sufficient zero credentials when `k=2`, I am in between some family related things, when I have a few quiet moments to think about how to explain I will suggest some sort of comment to clarify the `k*(k-1)` thing later

Reply:
@molnard you need this: https://github.com/zkSNACKs/WalletWasabi/pull/5611

Reply:
FWIW I strongly disagree, see #5611 for more discussion

Reply:
Two requests not looking good but it is easy and simple as hell.

Reply:
it's only necessary/correct do that if k=3, for k=2 only one request is needed

Reply:
ACK.